src/transforms/stdnames.jl

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(:spec)
+
+StdNames transform standardizes the column names.
```

Comment:
Please improve the docstring and write a text that helps end users understand the options. Try to follow the format of other docstrings.